### ISSUE TITLE
[Callout] - Imports and Behaviour broken in version >5.3.0

### ISSUE THREAD

**Issue by lrecknagel** [OfficeDev/office-ui-fabric-react#3038]
### Bug Report

- __Package version(s)__: >5.3.0
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Maybe

Requested priority: Blocking

Products/sites affected: internal app

#### Describe the issue:

When using version > 5.3.0 and try to implement a callout it breaks.
Importing Callout & DirectionalHint as follows results in an undefined Object for DirectionalHint:

```js
import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
// OR
import { Callout } from 'office-ui-fabric-react/lib/Callout';
import DirectionalHint from 'office-ui-fabric-react/lib/common/DirectionalHint';
```

Maybe this is related to: https://stackoverflow.com/questions/28818849/how-do-the-different-enum-variants-work-in-typescript

I tried to set a number of the DirectionalHint enum directly (rightCenter = 12), but the view is still broken.
This misalignment of the tooltip callout also causes issues with the onMouseEnter and onMouseLeave triggers.

#### Actual behavior:

https://codesandbox.io/s/v190071xyy

#### Expected behavior:

Should work like in version 5.3.0 and below.
https://codesandbox.io/s/73ox7z5z4q

### If applicable, please provide a codepen repro:

Broken version: https://codesandbox.io/s/v190071xyy
Working version: https://codesandbox.io/s/73ox7z5z4q

**Comment by dzearing** [comment]
@joschect This seems related to your recent changes! Please investigate asap.

Codepen as well:

https://codepen.io/dzearing/pen/bojpgX

**Comment by dzearing** [comment]
Nevermind. Figured it out. `target` should be used. Not `targetElement`, which has been deprecated for a long time and was removed (accidentally) in a recent build which should have been released with the 5.0 bump but missed. 

Forked your example and replaced `targetElement` with `target` and it works. Sorry for the confusion!
https://codesandbox.io/s/zl05nm2nyp

**Comment by lrecknagel** [comment]
Thanks for the investigation! This solves the problem. Overseen that prop name change.
